src/main/scala/chisel3/annotations/annotations.scala

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.annotations
```

Comment:
This seems like it probably should go into the `circt` package unless these different lowering conventions are added to the firrtl spec.

Reply:
Done, thanks!

Reply:
I'm not sure I agree with this. Currently there are _no_ Annotations enumerated in the spec, yet we have Annotations scattered all over.

Starting to put these in a common area would greatly simplify the structure of the code base. (And I would like to move all annotations here in the future.)

Reply:
I guess this is a nudge from me for the SiFive team to start putting interface constraints, i.e., the "ABI" of the circuit, into the firrtl spec.

Reply:
there is a PR open here for the current version: https://github.com/chipsalliance/firrtl-spec/pull/70